### PR TITLE
newer Intel Raptor Lake cputype in PerfCounters_x86

### LINKED ISSUE #3565: Intel CPU type 0xb06a0 unknown
Similar to https://github.com/rr-debugger/rr/issues/2614.

```
➜ rr record ls                         
[FATAL src/PerfCounters_x86.h:123:compute_cpu_microarch()] Intel CPU type 0xb06a0 unknown
```

```
➜  rr git:(master) lscpu
Architecture:            x86_64
  CPU op-mode(s):        32-bit, 64-bit
  Address sizes:         46 bits physical, 48 bits virtual
  Byte Order:            Little Endian
CPU(s):                  20
  On-line CPU(s) list:   0-19
Vendor ID:               GenuineIntel
  Model name:            13th Gen Intel(R) Core(TM) i9-13900H
    CPU family:          6
    Model:               186
    Thread(s) per core:  2
    Core(s) per socket:  14
    Socket(s):           1
    Stepping:            2
    CPU max MHz:         5400.0000
    CPU min MHz:         400.0000
    BogoMIPS:            5990.40
    Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_t
                         sc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf tsc_known_freq pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 sdbg
                          fma cx16 xtpr pdcm sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb ssbd ibrs ibpb sti
                         bp ibrs_enhanced tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc_adjust bmi1 avx2 smep bmi2 erms invpcid rdseed adx smap clflushopt clwb intel_pt sha
                         _ni xsaveopt xsavec xgetbv1 xsaves avx_vnni dtherm ida arat pln pts hwp hwp_notify hwp_act_window hwp_epp hwp_pkg_req hfi umip pku ospke waitpkg gfni vaes vpcl
                         mulqdq tme rdpid movdiri movdir64b fsrm md_clear serialize pconfig arch_lbr ibt flush_l1d arch_capabilities
Virtualization features: 
  Virtualization:        VT-x
Caches (sum of all):     
  L1d:                   544 KiB (14 instances)
  L1i:                   704 KiB (14 instances)
  L2:                    11.5 MiB (8 instances)
  L3:                    24 MiB (1 instance)
NUMA:                    
  NUMA node(s):          1
  NUMA node0 CPU(s):     0-19
Vulnerabilities:         
  Itlb multihit:         Not affected
  L1tf:                  Not affected
  Mds:                   Not affected
  Meltdown:              Not affected
  Mmio stale data:       Not affected
  Retbleed:              Not affected
  Spec store bypass:     Mitigation; Speculative Store Bypass disabled via prctl
  Spectre v1:            Mitigation; usercopy/swapgs barriers and __user pointer sanitization
  Spectre v2:            Mitigation; Enhanced IBRS, IBPB conditional, RSB filling, PBRSB-eIBRS SW sequence
  Srbds:                 Not affected
  Tsx async abort:       Not affected
```

### PR DESCRIPTION
**Pull request by GitMensch**
fixes #3565

### CODE REVIEW

File: src/PerfCounters_x86.h [hunk 1]
```diff
@@ -85,6 +85,7 @@ static CpuMicroarch compute_cpu_microarch() {
     case 0x906a0:
       return IntelAlderlake;
     case 0xb0670:
+    case 0xb06a0: // i9-13900H (Raptor Lake-H) / i7-1355U (Raptor Lake-U)
```
Comment (jtunhag) [inline src/PerfCounters_x86.h:88]: Goes for my `i7-13700H` too, so maybe skip the comment like for the rest of them (or add this too)?

Comment (khuey) [reply src/PerfCounters_x86.h:88]: Yeah we don't need this comment.

Comment (GitMensch) [reply src/PerfCounters_x86.h:88]: we do have it below... I'll drop the specific cpus

Comment (GitMensch) [reply src/PerfCounters_x86.h:88]: So @jtunhag What is your test result with the updated rev (or this change applied to master yourself)?

Comment (khuey) [reply src/PerfCounters_x86.h]: We have them for AMD because AMD has that weird logic for `ext_family`. Intel is pretty trivial by comparison.

**Comment by GitMensch** [comment]
Is there anything you want me to adjust before this can be merged?

**Comment by GitMensch** [comment]
I don't understand this, we have a comment for nearly all AMD and another Intel...
But as this is a single additional case, please just add it yourself instead of the additional merge work.

**Pull request closed without merging**